core/sqf/sql/scripts/cleanlogs

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#
+# @@@ START COPYRIGHT @@@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# @@@ END COPYRIGHT @@@
+
+# This script cleans up the log files of various components
+if [ -e $SQ_PDSH ]; then
+   L_PDSH="/usr/bin/pdsh $MY_NODES "
+else
+   L_PDSH=
+fi
+
+function usage() {
+   prog=`basename $0` 
+   echo ""
+   echo "$prog { all | dcs | rest | core }"
+   echo "    all  --- Remove files from core, dcs and rest logs folder"
+   echo "    dcs  --- Remove files from $DCS_INSTALL_DIR/logs folder"
+   echo "    rest --- Remove files from $REST_INSTALL_DIR/logs folder"
+   echo "    core --- Remove log files residing in $MY_SQROOT/logs folder"
```

Comment:
it's might be helpful that save them before remove, or give users some tips and tell them the logs will be removed. if they insist on, go ahead to remove them. yes, it's a utility, but some users might use it, right? we know what they are in MY_SQROOT/logs folder, but the users may not, so those info should be included. otherwise, users may make mistakes.